bedrock/settings/base.py

```diff
@@ -1313,3 +1314,34 @@ def get_default_gateway_linux():
 
 # Issue 7508 - Convert.com experiment sandbox
 CONVERT_PROJECT_ID = ('10039-1003350' if DEV else '10039-1003343')
+
+# Mozilla VPN Geo restrictions
+# https://github.com/mozilla-services/guardian-website/blob/master/server/constants.ts
+VPN_ALLOWED_COUNTRY_CODES = [
+  'CA', #Canada
+  'MY', #Malaysia
+  'NZ', #New Zealand
+  'SG', #Singapore
+  #United Kingdom + "Territories"
+  'GB', #United Kingdom of Great Britain and Northern Island
+  'GG', #Guernsey (a British Crown dependency)
+  'IM', #Isle of Man (a British Crown dependency)
+  'IO', #British Indian Ocean Territory
+  'JE', #Jersey (a British Crown dependency)
+  'UK', #United Kingdom
+  'VG', #Virgin Islands (British)
+  #USA + "Territories"
+  'AS', #American Samoa
+  'MP', #Northern Mariana Islands
+  'PR', #Puerto Rico
+  'UM', #United States Minor Outlying Islands
+  'US', #United States of America
+  'VI', #Virgin Islands (U.S.)
+]
+
+# Mozilla VPN subscription prices
+VPN_PRICE_MONTHLY = {
+    'US': {
+        'monthly': 'US$4.99'
```

Comment:
TIL: 
<img width="681" alt="image" src="https://user-images.githubusercontent.com/2692333/108175266-2d40f780-70c6-11eb-8f14-9234a7feb833.png">

_I was bugging about the lack of a space between `US` and `$` and wanted to confirm this is noted correctly!_

Reply:
Yeah it reads strange to me too, but for now it's as correct as can be. In the near future we'll be adding more currencies, so I'd like to revisit this wording when we get to localization.